package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@movable/tailwind-config",
-  "version": "2.1.0",
+  "version": "2.2.0-alpha.1",
```

Revert this. Needed only for release
```suggestion
  "version": "2.1.0",
```